### DISCUSSION TITLE
《亮剑》 | /posts/drawing-sword/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#629] [Announcements]
https://yuanfan.rbind.io/posts/drawing-sword/

**Comment by XiangyunHuang** [comment]
看来你看的是一个完整版的《亮剑》，小说的深度往往比影视剧好。

**Reply by earfanfan** [reply]
我只看了电视剧，没看小说。“小说的深度往往比影视剧好”这个结论是根据这一个样本得到的么，其实也有别的，比如《甄嬛传》电视剧。

**Reply by XiangyunHuang** [reply]
你可以试着先看小说，再看影视剧。

**Reply by XiangyunHuang** [reply]
我觉得《白鹿原》是最典型的一部作品，告诉我们小说有多好，目前拍的影视剧又有多烂。

**Reply by earfanfan** [reply]
《白鹿原》我听说过，知道有小说，也有改编的电影电视剧，不过都没看过。目前拍的影视剧……你最近有看到撒好看的电视剧没，有没有觉得可以安利给我也看看的？没有也没关系，反过来让我给你安利我也想不出来，哈哈哈。

额，其实刚刚本来我的杠精性格又要发作了，我本来想说单纯比较小说和电视剧这两种内容表达形式的话，只因形式决定优劣的话片面了些……不过又仔细想了下，其实你留言的文字隐去了一个前提条件，湘云你本来想说的是不是针对先有小说后改编电视剧的情况？又不过，我想起来你之前夸过《繁花》，这部电视剧也是由小说改编来的，不过我也都没看过。

**Reply by XiangyunHuang** [reply]
高华老师的课程和亮剑莫名的相合。李云龙、赵刚、丁伟、田墨轩分别是代表农民阶层、知识分子（投身革命）、知识分子（民主党派）在讲话。

《繁华》是另一个例外，王家卫太厉害了，小说是一个意思，电视剧又是另一个意思，两个意思都很好，我是看原作的采访知道的。

**Comment by meeeeeeeeo** [comment]
我这些天也在看亮剑，每次看之前都会先切到第六集看一句：“嘿，你他娘的还真是个天才”

**Reply by earfanfan** [reply]
咦，真有意思，原来还可以这样操作呀。要是看到李云龙也夸我是个天才，我也感到很高兴。

**Reply by meeeeeeeeo** [reply]
说回看剧的话，大明王朝也很不错，借古喻今，借虚喻实，博主有看过不

**Reply by earfanfan** [reply]
不瞒你说，我看完《亮剑》以后，紧接着看的下一部电视剧就是《大明王朝1566》，这部剧我好几年前就看了二十几集但是没坚持看完，这次又从头看。可是，看到第二集我就又看不下去了，那个严肃沉重的氛围，会让我容易感到心塞。

**Reply by earfanfan** [reply]
我现在脑子里印象特别深刻的是，第一集里内阁会议，张居正说财政上要拨款给戚继光他们，整顿军务，肃清海运，然而严嵩打断张居正，说这件事张和他商议过，然后立马就把话题转到改稻为桑，明明张是主张通过对外贸易来增加财政收入一千万两，结果被严嵩一说分分钟变成用养蝉织布卖丝绸来增加财政收入一千万两。唉，总之，这第一集就把我看得心塞了，因为我已经知道改稻为桑后面会怎样变成迫害老百姓的事。

**Reply by meeeeeeeeo** [reply]
大明王朝里面，演绎了很深的权力斗争，确实非常沉重，普通老百姓在剧里剧外连浮萍都算不上。
但是我看剧和回顾的时候的感受是，现实的沉重和崇高的理想都是社会中正常存在的东西，心中留有崇高的理想，经过剧中沉重氛围的洗礼，才能更显珍贵

**Reply by earfanfan** [reply]
嗯，谢谢你分享的感受，我觉得你说得非常有道理，所以我立刻决定今天下班回去就换种心态接着看，如果这一次坚持看完了，就再耕一篇博客。

对了，如果你有看书的习惯的话，我想安利你看看吴毅老师写的《小镇喧嚣》，书里记录的“逼民致富”和剧里的“改稻为桑”实在是非常相似。

**Reply by meeeeeeeeo** [reply]
感谢推荐，有时间买来看下
